public/index.html

```diff
@@ -92,6 +93,36 @@ <h1>TriMet TravelMap</h1>
 						</div>
 					</div>
 				</div>
+
+				<!-- Routes sidebar; contains subpages for routes and stops for
+					each route in both directions. -->
+				<div id="stops-sidebar" data-tab="stops" data-tablist="stops-pages">
+					<!-- Routes subpage for showing the list of routes. -->
+					<div data-tab="stops">
+						<div id="stops-header">
+							<input id="stops-search" type="search" placeholder="Search for stop (hit enter to search)" />
+							<button id="stops-clear-search" class="special light-button">&#xD7;</button>
+						</div>
+						<div id="stops-list">
+							<div class="loading">Loading stops...</div>
+						</div>
+					</div>
+
+					<!-- Routes subpage for showing the list of stops in a
+						route in both directions. -->
+					<div data-tab="transport">
+						<div id="transport-header">
+							<button id="transport-back" class="special light-button">&#x2190;</button>
+							<div id="transport-chosen"></div>
+						</div>
+						<div class="selector" data-tablist="transport-type">
+							<button id="transport-selector" class="special" data-tab="transport-0"></button>
+						</div>
+						<div id="transport-lists" data-tablist="transport-type">
+							<div id="transport-0-list" data-tab="transport-0"></div>
+						</div>
```

Comment:
The stops don't have tabs, so this can all be replaced with a single div `<div id="transport-list"></div>`.  The CSS will need a bit of updating.